app/templates/cof_r2w2_all_questions.html

```diff
@@ -7,7 +7,11 @@
 {% block content %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-        <span class="govuk-caption-l">Community Ownership Fund {{ round_title }}</span>
+            {% if round_data.title|length %}
```

Comment:
Why are we checking the length if we don't then use `round_data.title`?

Reply:
Because we're being tolerant of the round call failing and the round_data being empty - if its empty, we want to fail gracefully so we check if the title is set - if it isn't we just dont render that content rather than bring the whole service down

Reply:
But why not just hard code `Community Ownership Fund Round 2 Window 2` regardless of whether title is there? It's hardcoded if it's there (line 11)

Reply:
Your dead right, I'd only fixed half of it after our chat before.